src/server.c

```diff
@@ -5032,6 +5033,7 @@ sds genRedisInfoString(const char *section) {
             "expired_time_cap_reached_count:%lld\r\n"
             "expire_cycle_cpu_milliseconds:%lld\r\n"
             "evicted_keys:%lld\r\n"
+            "evicted_time_cap_reached_count:%lld\r\n"
```

Comment:
I think "eviction_time_exceeded_count" is clearer, but I see that it is emulating the "expired_time_cap_reached_count". If we want to stay with consistency, it should still be "eviction_time_cap_reached_count", but I would still prefer my original suggestion.

Reply:
I keep consistent with `expired_time_cap_reached_count`. But in fact I think `expired_time_cap_reached_count` is not clear. `eviction_time_exceeded_count` is more clearer.

Reply:
I thought expired_time was weird because the time is not expired which is the only meaning I can parse from expired_time. I instead used an attributive noun, eviction, to better define "time" since it defines the context of a noun (it's time used with evictions). I'm a native english speaker though so I would use whatever seems clearest to you.

Reply:
updated.